config/settings/base.py

```diff
@@ -421,4 +423,7 @@ def env(name, default=NoDefaultValue, type_=str):
 SENTRY_DSN = env("SENTRY_DSN", default="")
 
 if SENTRY_DSN:
-    sentry_sdk.init(dsn=SENTRY_DSN, integrations=[DjangoIntegration()])
+    sentry_sdk.init(
+        dsn=SENTRY_DSN,
+        integrations=[DjangoIntegration(), RedisIntegration(), RqIntegration()],
```

Comment:
Hm, it's not clear to me whether the `RqIntegration` needs a separate `.init()` call within the worker process?

https://docs.sentry.io/platforms/python/rq/

Reply:
Settings should be one of the first things loaded, so I think this will `.init` it properly. I hope at least!